server/taskIngest.go

```diff
@@ -88,10 +88,46 @@ func ingestBillingDataForBillRepository(ctx context.Context, aaId, brId int) (er
 			"error":            err.Error(),
 		})
 	}
+	updateIdentityAndSubAccounts(ctx, aa)
```

Comment:
Shoudn't we updateIdentity when creating or editing an AWS Account as well?

Reply:
It is done when creating, edit can only change pretty and payer so we don't need to check at this moment. And this is done here to add identity for old aws accounts